Documentation/Tools/OptimizeWindow.md

```diff
@@ -1,6 +1,6 @@
 # Optimize window
 
-The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing.
+The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing. The utility is included with the `Microsoft.MixedReality.Toolkit.Unity.Tools` package.
```

Comment:
Does our package have `.Unity` in the name?

Reply:
```suggestion
The MRTK Optimize Window is a utility to help automate and inform in the process of configuring a mixed reality project for best [performance](../Performance/PerfGettingStarted.md) in Unity. This tool generally focuses on rendering configurations that when set to the correct preset can save milliseconds of processing. The utility is included with the `Microsoft.MixedReality.Toolkit.Tools` package.
```

Reply:
Yes. "Microsoft.MixedReality.Toolkit.Unity.Tools.2.5.3.unitypackage"

Reply:
Ah! interesting. Should we clarify UPM vs .unitypackage, since they have different names?